frontend/src/components/auth/Login.jsx

```diff
@@ -90,6 +92,56 @@ export default function Login() {
         <ContentSection>
           <Row>
             <h3 className="fw-bold col-auto me-auto mt-2">Log In</h3>
+            <div className="col-auto">
+              <a
+                href={`${AUTH_BASE_URI}/gihutb`}
+                onClick={(e) => {
+                  e.preventDefault();
+                  const url = `${AUTH_BASE_URI}/github`;
+                  axios
+                    .get(`${url}?no_redirect=true`)
+                    .then(() => {
+                      window.location = url;
+                    })
+                    .catch((error) => {
+                      if (
+                        error?.response?.status === 401 &&
+                        error.parsedMsg.includes("OAuth is not configured.")
+                      )
+                        addToast(
+                          "Login failed!",
+                          "OAuth is not configured. " +
+                            "Check documentation to set it up.",
+                          "danger",
+                          true
+                        );
+                      else throw error;
+                    });
+                }}
+              >
+                <img
+                  src={`${PUBLIC_URL}/icons8-github-48.png`}
```

Comment:
in the dark mode of IntelOwl, I think it is better to use the icon with opposite colors (white as main background)